net/stun/stun.go

```diff
@@ -137,15 +137,15 @@ func foreachAttr(b []byte, fn func(attrType uint16, a []byte) error) error {
 		}
 		attrType := binary.BigEndian.Uint16(b[:2])
 		attrLen := int(binary.BigEndian.Uint16(b[2:4]))
-		attrLenPad := attrLen % 4
+		attrLenPad := (attrLen + 3) &^ 3
```

Comment:
I'd prefer this be more explicit. The 3s and &^ bitmask is a bit arcane.

Let's just do:

```go
padLen := 4 - (attrLen % 4)
totalLen := attrLen + padLen
```

Then both the padding computation and the variable names are explicit.

Reply:
A new test addition would also be good.

Reply:
Your suggestion is incorrect.

It'll add 4 pad bytes when not needed.